### PR TITLE
Fix build by adding libacl and libncurses

### PR DESCRIPTION
**Pull request by elBoberido**
## Pre-Review Checklist for the PR Author

1. [x] Code is formatted with clang-format
1. [x] Update the PR title
   - Follow the same conventions as for commit messages
   - Link to the relevant issue
1. [x] Relevant issues are linked
1. [x] Add sensible notes for the reviewer
1. [x] All checks have passed
1. [x] Assign PR to reviewer

## Checklist for the PR Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code according to our coding style and naming conventions
- [x] Unit tests have been written for new behavior
- [x] Copyright owner are updated in the changed files
- [x] PR title describes the changes

## Post-review Checklist for the PR Author

1. [x] All open points are addressed and tracked via issues

## References

- none

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,5 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - run: cmake -B build && cmake --build build
-
+      - name: Install dependencies
+        run: apt-get update && apt-get install -y libacl1-dev libncurses5-dev
```
Comment (dkroenke) [inline .github/workflows/build.yml]:
```suggestion
        run: sudo apt-get update && apt-get install -y libacl1-dev libncurses5-dev
```

**Review by mossmaurice — APPROVED** [review]

**Review by mossmaurice — CHANGES_REQUESTED** [review]
https://img.devrant.com/devrant/rant/r_1902819_KMYZA.jpg

**Comment by mossmaurice** [comment]
> Commits are properly organized and messages are according to the guideline

Do we follow the usual guidelines in this repo, too?

**Review by dkroenke — APPROVED** [review]

**Comment by elBoberido** [comment]
> > Commits are properly organized and messages are according to the guideline
> 
> Do we follow the usual guidelines in this repo, too?

@mossmaurice yes, I squashed the `Fix fix` and `Fix fix fix` after the fix worked

**Review by mossmaurice — APPROVED** [review]

**Pull request merged**